dist/index.c2b3c8ae.css

```diff
@@ -0,0 +1,71 @@
+#qg-two-col-aside.wide #qg-primary-content {
```

Comment:
I'm really not a fan of having 'dist' binaries commited into a source repo.

@asifamingov , please raise a ticket for making a matrix-projects-bin repo and cloning the formio repo pipeline for 'fleshing' it out via github action calls as well as make progression to reference that repo in squiz matrix etc.

Reply:
'Dist' folder is temporary I will create a Gitbridge.